qemu/tests/iperf_test.py

```diff
@@ -86,6 +87,19 @@ def is_iperf_running(name_pattern, session=None):
                                     ignore_status=True, verbose=False)
         return status == 0
 
+    def rss_test():
+        logging.info('Run the command "netkvm-wmi.cmd rss" to collect statistics')
+        if params.get("rss_test", "no") == "yes":
+            rss_test_cmd = utils_misc.set_winutils_letter(
+                guest_session, params["rss_test_cmd"])
+            output = guest_session.cmd(rss_test_cmd)
+            value1 = int(output.splitlines()[1].split()[-4])
+            value2 = int(output.splitlines()[1].split()[-2])
+            if value1 == 0 & value2 == 0:
+                logging.info("Rss support for virtio-net driver is works well")
+            else:
+                test.fail("Rss support for virtio-net driver is bad")
```

Comment:
```suggestion
            if not (value1 == 0 and value2 == 0):
                test.fail("Rss support for virtio-net driver is bad")
            logging.info("Rss support for virtio-net driver is works well")              
```

Reply:
Thank you for your help, I have solved all the above problems.